hledger-lib/Hledger/Read/JournalReader.hs

```diff
@@ -467,7 +466,12 @@ periodictransactionp = do
   -- a period expression
   pos <- getPosition
   d <- liftIO getCurrentDay
-  (periodtxt, (interval, span)) <- lift $ first T.strip <$> match (periodexprp d)
+  periodExcerpt <- lift $ excerpt_ singlespacedtextp
```

Comment:
With this code, two spaces enforces the separation of period expression and description. But if you forget and write one space, and are unlucky with wording, it's still possible to get unintended overparsing. Eg `~ monthly In 2019 we will change this` is parsed as `monthly In 2019`. 

Which will be simplest and least surprising overall - allow one space, but document the above and recommend two spaces for robustness; or require two spaces ? I lean towards the latter. Any other options ?

Reply:
Another option: instead of english-like period expressions, use some more distinctive and complete syntax like https://en.wikipedia.org/wiki/ISO_8601#Repeating_intervals . Dropping the old syntax is probably out of the question though.